java-client/src/main/java/org/scassandra/http/client/UnpreparedConfig.java

```diff
@@ -0,0 +1,32 @@
+package org.scassandra.http.client;
+
+import java.util.Map;
+
+import com.google.common.collect.ImmutableMap;
+import org.scassandra.server.priming.ErrorConstants;
+
+public class UnpreparedConfig extends Config {
```

Comment:
Hadn't seen this error, perhaps I should use it rather than throw an error when scassandra actually hasn't got a prepare.